index.html

```diff
@@ -0,0 +1,206 @@
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section id="faq" style="padding-top: 100px;">
+            <div style="text-align: center;">
+                <h2 style="font-size: 40px;">FAQ</h2>
+            </div>
+            <div  style="display: flex; justify-content: center; ">
+                <ul  style="list-style: none;">
+                    <li style="padding: 10px;">
+                        <details>
+                <summary  style="list-style: none; font-size: 24px;">Quienes pueden ordenar el servicio?</summary>
+                <p>Debe ser una persona mayor de 18 años</p>
+              </details>
+                    </li>
+                    <li style="padding: 10px;">
+                        <details>
+                <summary  style="list-style: none; font-size: 24px;">Se puede ordenar el servico en cualquier parte del estado?</summary>
+                <p>El servicio solo se brinda en los municipios de Mixquiahuala, Actoapan, Progreso, Pachuca</p>
+              </details>
+                    </li>
+                    <li style="padding: 10px;">
+                        <details>
+                <summary  style="list-style: none; font-size: 24px;">El servicio se brinda cualquier día de la semana?</summary>
+                <p>Así es cualquier dia que lo quieras contratar</p>
+              </details>
+                    </li>
+                    <li style="padding: 10px;">
+                        <details>
+                <summary  style="list-style: none; font-size: 24px;">Puedo cancelar el servicio si ya lo ordené?</summary>
+                <p>Si claro, pero solo te reembolsamos el 15% de tu anticipo debido a que se hacen las compras necesarias para dar el servicio</p>
+              </details>
+                    </li>
+                    <li style="padding: 10px;">
+                        <details>
+                <summary  style="list-style: none; font-size: 24px;">Tiene un limite de tiempo una vez que se comienza a dar el servicio?</summary>
+                <p>Solo hasta que se termine el producto que se presupuestó</p>
+              </details>
+                    </li>
+                </ul>
+            </div>
+        </section>
+    </main>
+
+    <footer id="contact" style="padding-top: 20px;">
+        <div style="display: flex; justify-content: center; ">
+            <div style="padding: 10px 350px 10px 10px;">
+                <div>
+                    <h4>informacion de contacto</h4>
+                </div>
+                <div>
+                    <label>Ubicacion: </label>
+                    <a>AV centro al lado de la pizzeía george</a>
+                </div>
+                <div>
+                    <label>Telefono: </label>
+                    <a>1234567891</a>
+                </div>
+                <div>
+                    <label>Correo electronico: </label>
+                    <a>elatoron@gmail.com</a>
```

Comment:
```suggestion
                    <a href="mailto:elatoron@gmail.com">elatoron@gmail.com</a>
```
Para que se le pueda dar click para enviar un email